tests/quantization/quanto/test_quanto.py

```diff
@@ -0,0 +1,177 @@
+import unittest
+
+import torch
+
+from diffusers import (
+    QuantoConfig,
+)
+from diffusers.models.transformers.transformer_flux import FluxTransformer2DModel
+from diffusers.utils import is_optimum_quanto_available
+from diffusers.utils.testing_utils import (
+    nightly,
+    numpy_cosine_similarity_distance,
+    require_accelerate,
+    require_big_gpu_with_torch_cuda,
+    torch_device,
+)
+
+
+if is_optimum_quanto_available():
+    from optimum.quanto import QLinear
+
+
+@nightly
+@require_big_gpu_with_torch_cuda
```

Comment:
Are we adding this marker because of the `torch.compile()` test? Otherwise, the tests seem fine to be executed without the marker.